.github/workflows/ci.yml

```diff
@@ -58,6 +58,8 @@ jobs:
         with:
           path: "**/node_modules"
           key: ${{ runner.os }}-modules-${{secrets.ACTIONS_CACHE_VERSION}}-${{ hashFiles('yarn.lock') }}
+      - name: Generate code
```

Comment:
Could we use something more precise? Like 
`Generate graphql schema, mutations and query types`

Reply:
As long as the script being called is named `codegen`, I'd prefer the name to match it.
Since you've asked for some sort of disambiguation, I have modified an existing line calling `codegen:ci` however to denote the fact that it also checks for uncommitted changes after generating the source code.